notebooks/official/generative_ai/text_embedding_api_semantic_search_with_scann.ipynb

```diff
@@ -0,0 +1,1153 @@
+{
```

Comment:
<p>Each code cell should have text cell explaining what you are doing.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2127/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>